docs/topics/auth/passwords.txt

```diff
@@ -228,3 +228,16 @@ from the ``User`` model.
 
    Checks if the given string is a hashed password that has a chance
    of being verified against :func:`check_password`.
+
+Passwords are sent in plain text to your server unless you use HTTPS
+====================================================================
+
+You should always use :ref:`HTTPS <security-recommendation-ssl>` for your site.
+This is especially true if you require the user to send any sensitive
+information using a HTML form, such as login details. With plain HTTP, the
+password will be sent in plaintext from the user's computer to your server,
+making it vulnerable to 'password sniffing'. With :ref:`HTTPS
```

Comment:
Can you wrap `password sniffing` in `*` (italics) instead: `"... vulnerable to *password sniffing*. With ..."`